libs/@local/hash-graph-types/graph_types/base.py

```diff
@@ -0,0 +1,57 @@
+"""Base classes for all graph types created by `.create_model()`.
+
+These classes are primarily used as markers
+"""
+from abc import ABC
+from typing import ClassVar
+
+from pydantic import BaseModel, GetJsonSchemaHandler
+from pydantic.json_schema import JsonSchemaValue
+from pydantic_core import CoreSchema
+
+
+class TypeInfo(BaseModel):
+    """Information about a type."""
+
+    identifier: str
+    schema_url: str
+    title: str
+    description: str | None = None
+    kind: str
+
+
+class Type(BaseModel, ABC):
+    """Base class for all graph types."""
```

Comment:
This is only true for ontology types.

Reply:
fixed in [d2b08ad](https://github.com/hashintel/hash/pull/2776/commits/d2b08adb062fdb15c189843dbe796342ebe79776)